src/Microsoft.ML.Mkl.Components/SymSgdClassificationTrainer.cs

```diff
@@ -19,20 +19,20 @@
 using Microsoft.ML.Trainers;
 using Microsoft.ML.Transforms;
 
-[assembly: LoadableClass(typeof(SymbolicSgdTrainer), typeof(SymbolicSgdTrainer.Options),
+[assembly: LoadableClass(typeof(SymbolicSgdLogisticRegressionBinaryTrainer), typeof(SymbolicSgdLogisticRegressionBinaryTrainer.Options),
```

Comment:
>SymbolicSgdLogisticRegressionBinaryTrainer [](start = 32, length = 42)

`LogisticRegressionBinaryTrainer` => `LbfgsLogisticRegressionTrainer`. Shall `SymbolicSgdBinaryTrainer` => `SymbolicSgdLogisticRegressionTrainer` instead of `SymbolicSgdLogisticRegressionBinaryTrainer`? Right now, the namings don't line up.

Reply:
@rogancarr..could you cross-check ? To me the *Class* names do match up `LbfgsLogisticRegressionBinaryTrainer` and `SymbolicSgdLogisticRegressionBinaryTrainer`

---
In reply to: [268859710](https://github.com/dotnet/machinelearning/pull/3034#discussion_r268859710) [](ancestors = 268859710)

Reply:
Yeah. Our conclusion was having `LR` and `Binary` for trainer classes and only `LR` for APIs.

---
In reply to: [268865148](https://github.com/dotnet/machinelearning/pull/3034#discussion_r268865148) [](ancestors = 268865148,268859710)

Reply:
Got it. I was mixing up catalogs & classes.